tests/managers_regress/models.py

```diff
@@ -147,3 +147,70 @@ class RelationModel(models.Model):
 
     def __str__(self):
         return force_text(self.pk)
+
+
+class BaseModel(models.Model):
```

Comment:
@charettes, I will adjust my pull request and place all models into tests methods with `managed=False` and using `isolate_apps `. Thank you.
@timgraham, please explain where should I put models since they're in separate file models.py now and according to what @charettes says I should place them into tests methods. Thank you.